client/game/GameView.svelte

```diff
@@ -28,21 +29,49 @@
 	}
 
 	socket.on("roomStarted", function(data) {
-		loadRoom(data);
 		started = true;
+		loadRoom(data);
 	});
 
 	socket.on("joinedSuccessfully", function(data) {
 		hasJoined = true;
 		loadRoom(data);
 	});
+
+	setInterval(() => {
+		const cards = [randomCard(), randomCard(), randomCard()];
+		plays = [
+			{
+				cards: cards,
+				player: "test",
+				valid: isValidPlay(cards),
+				id: Math.random()
+			},
+			...plays.slice(0, 10)
```

Comment:
why 10? Can we get a constant for this number so that it's more legible?

Reply:
It is just to prevent the array from getting too long and arbitrary. Do you think I should still move it to a variable?

Reply:
Yeah, I still think we should move it to a variable. Otherwise, if we come back, we won't know why it's `10` instead of `11` (the number of cards in a hand)